### PR TITLE
Simplify code

### PR DESCRIPTION
**Pull request by oscarotero**
Hi.
Thanks for creating this template for Lume. I want to propose the following changes:

- Simplify `_config.js` file:
  - The `port` variable can be customized using the `--port` CLI option. No need to define it in this way.
  - There's no a `CHANGELOG.md` file, so no need to be ignored
  - The `node_modules` folder is ignored by default by Lume.
- Simplify `package.json` file:
  - Fixed Lume name in the package name.
  - Due this is a Deno project, I think the only dependency should be deno-bin. I don't know why the `serve` dependency is needed.
  - I even try to remove the `devDependencies` key and call directly `npx deno-bin task build` but it makes the deploy to fail, so I left it as is.

### CODE REVIEW

File: _config.js [hunk 1]
```diff
@@ -1,12 +1,7 @@
 import lume from "lume/mod.ts";
 
-const port = Deno.env.get("PORT") ? Number(Deno.env.get("PORT")) : 8000;
-const site = lume({
-    server: {
-        port: port
-    }
-});
+const site = lume();
```
Comment (palmiak) [inline _config.js:3]: I would leave it as it was. Why? Passing the `PORT` through the CLI during deployment is sometimes flaky on our platform, but this solution always worked.

File: package.json [hunk 2]
```diff
@@ -1,17 +1,13 @@
 {
-  "name": "hello-world-lumen",
+  "name": "hello-world-lume",
   "version": "1.0.0",
   "description": "",
   "main": "_config.js",
   "scripts": {
-    "build": "deno task lume",
-    "start": "serve _site/"
```
Comment (palmiak) [inline package.json:8]: We aim to make our starters work with both Static Site Hosting and Application Hosting. App hosting requires `serve` to work correctly.

But you are right - if not the App hosting, it wouldn't be required

**Review by palmiak — CHANGES_REQUESTED** [review]
Hey

Thanks for this PR and the explanation. It does help a lot ❤️ 

I added some comments - the `PORT` and `serve` parts are here to make it work smoothly with all our services.

File: _config.js (hunk 1 above)
Comment (oscarotero) [reply _config.js:3]: The port is used only during local development (when running the local server and watcher with `deno task serve`). It's not used for building or serving the site from kinsta servers. At least this is what I could see [in this testing repo](https://github.com/oscarotero/testing-kinsta/tree/master) that is served correctly here https://testing-kinsta-gufs7.kinsta.page/ without any configuration.

BTW, maybe the config should be `_config.ts` (in TypeScript), due it's the format majority choosen by Lume users.

**Pull request closed without merging**